server/mysterium_api.go

```diff
@@ -64,26 +64,27 @@ func (mApi *mysteriumAPI) RegisterProposal(proposal dto_discovery.ServiceProposa
 	return err
 }
 
-func (mApi *mysteriumAPI) NodeSendStats(nodeKey string, signer identity.Signer) error {
+func (mApi *mysteriumAPI) PingProposal(proposal dto_discovery.ServiceProposal, signer identity.Signer) error {
```

Comment:
New method name is false - it states that it acts on "proposal", but implementation acts on "provider".
Naming it `UpdateProviderStatus` seems ok for me.

Reply:
Tried: `RegisterProvider(proposal)` and `UpdateProviderStatus(proposal)` and `FindProviders(providerId)`, looks weird and reverted.

 I want to make driving change here:
```
 identity -> providerID   -> publish Proposal(providerID, serviceType, ID)
             -> consumerID -> consume Proposal(providerID, serviceType, ID)
```

Reply:
Method name should indicate what method does, not what you think it should do in the future.
>Tried: RegisterProvider(proposal) and UpdateProviderStatus(proposal) and FindProviders(providerId), looks weird and reverted.

Why `FindProviders(providerId)` ?? It finds proposals, not providers, so of course it looks weird :D